primitives/state-machine/src/trie_backend_essence.rs

```diff
@@ -438,6 +440,72 @@ where
 			false,
 		);
 	}
+
+	/// Check remaining state item to migrate. Note this function should be remove when all state
+	/// migration did finished as it is only an utility.
+	// original author: @cheme
+	pub fn check_migration_state(&self) -> Result<(u64, u64)> {
+		let threshold: u32 = 33;
```

Comment:
why not use sp_core::storage::DEFAULT_MAX_INLINE_VALUE ?

Reply:
Yes would be better, my fault, I initially write the code in a crate that did not have access to it, but now that it is in state machine it should clearly be use.

Reply:
updated.